bench/topic_perf.rb

```diff
@@ -219,6 +221,84 @@ def swift_select_title_id(l=1000)
 exit(-1) unless results.uniq.length == 1
 
 
+#Benchmark.ips do |r|
+#  r.report('string') do |n|
+#    while n > 0
+#      s = +''
+#      1_000.times { |i| s << i; s << i }
+#      n -= 1
+#    end
+#  end
+#  r.report('array') do |n|
+#    while n > 0
+#      1_000.times { |i| [i, i] }
+#      n -= 1
+#    end
+#  end
+#
+#  r.compare!
+#end
+
+# Comparison:
+#   array:    13041.2 i/s
+#  string:     4254.9 i/s - 3.06x  slower
```

Comment:
in next PR I suggest replaced code
```ruby
  s = +""
  Topic.limit(1000).order(:id).pluck(:id, :title).each do |id, title|
    s << id.to_s
    s << title
  end
  s
```

to 
```ruby
  Topic.limit(1000).order(:id).pluck(:id, :title).each do |id, title|
    [id, title]
  end
```

because use string add unnecessary overhead into benchmarks

Reply:
Keep in mind, the benchmark is meant to create the strings, the idea behind it is "What is the fastest way of generating the same exact string from multiple libraries.